tests/system/Database/Live/EscapeTest.php

```diff
@@ -72,7 +72,7 @@ public function testEscapeLikeString()
     public function testEscapeLikeStringDirect()
     {
         if ($this->db->DBDriver === 'MySQLi') {
-            $expected = "SHOW COLUMNS FROM brands WHERE column LIKE 'wild\_chars%'";
+            $expected = "SHOW COLUMNS FROM brands WHERE column LIKE 'wild\\_chars%'";
```

Comment:
Is this right? I'm not sure this should ever have been "escaped".

Reply:
This is still the same, though uncomfortable at first.

https://3v4l.org/lk32I